docker-storage-setup.sh

```diff
@@ -35,6 +35,39 @@ STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
 STORAGE_OUT_FILE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
+usage() {
+  cat <<-FOE
+    Usage: $1 [OPTIONS] INPUTFILE OUTPUTFILE
+
+    Grows the root filesystem and sets up storage for container runtimes
+
+    Options:
+      --help    Print help message
+      --reset   Reset your docker storage to init state. 
+FOE
+}
+
+OPTS=`getopt -o h -l reset -l help -- "$@"`
+eval set -- "$OPTS"
+reset=0
```

Comment:
All global variables are in capital letters in this script. lets continue to follow that convention.